proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -40,6 +40,38 @@ lemma current_time_bounded_strengthen:
   unfolding current_time_bounded_def
   by (erule order_trans[rotated], clarsimp)
 
+(* FIXME RT: move? *)
+lemma of_nat_helper:
```

Comment:
This one should be true for any word length. Could you generalise to `'a::len word` and move to Word_Lemmas_Internal so I can merge it upstream (happy for the move to be in a separate cleanup phase).

Reply:
Yep! Done (will include a commit with just this in the next push)